main/modes/games/picross/picross_menu.c

```diff
@@ -97,63 +97,347 @@ struct
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "toadstool",
+        .pzl         = AA_TOADSTOOL_PZL_WSG,
+        .slv         = AA_TOADSTOOL_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "donutjump",
+        .pzl         = AF_DONUTJUMP_PZL_WSG,
+        .slv         = AF_DONUTJUMP_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "picross",
+        .pzl         = AF_PICROSS_PZL_WSG,
+        .slv         = AF_PICROSS_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "superswadgeland",
+        .pzl         = AF_SUPERSWADGELAND_PZL_WSG,
+        .slv         = AF_SUPERSWADGELAND_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "swadgebros",
+        .pzl         = AF_SWADGEBROS_PZL_WSG,
+        .slv         = AF_SWADGEBROS_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "tiltrads",
+        .pzl         = AF_TILTRADS_PZL_WSG,
+        .slv         = AF_TILTRADS_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "galacticbrickdown",
+        .pzl         = CF_GALACTICBRICKDOWN_PZL_WSG,
+        .slv         = CF_GALACTICBRICKDOWN_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "lumberjacks",
+        .pzl         = CF_LUMBERJACKS_PZL_WSG,
+        .slv         = CF_LUMBERJACKS_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "magtroidpocket",
+        .pzl         = CF_MAGTROIDPOCKET_PZL_WSG,
+        .slv         = CF_MAGTROIDPOCKET_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "pushykawaiigo",
+        .pzl         = CF_PUSHYKAWAIIGO_PZL_WSG,
+        .slv         = CF_PUSHYKAWAIIGO_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "bigbug",
+        .pzl         = EF_BIGBUG_PZL_WSG,
+        .slv         = EF_BIGBUG_SLV_WSG,
+        .marqueeFact = "Dr. Ovo Garbotnik fought bugs in a landfill. \"--Incoming transmission from mr. dev\" What in "
+                       "the heck? \"James was trying to solve the design issues that Dave the Diver ran into. But he "
+                       "sort of just ran into the same problems.",
+    },
+    {
+        .name        = "pango",
+        .pzl         = EF_PANGO_PZL_WSG,
+        .slv         = EF_PANGO_SLV_WSG,
+        .marqueeFact = "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    },
+    {
+        .name        = "2048",
+        .pzl         = EF_2048_PZL_WSG,
+        .slv         = EF_2048_SLV_WSG,
+        .marqueeFact = "Did you know? 2048 is so named because that's The highest value, the dev was able to get to "
```

Comment:
Careful when copying text from other sources. Sometimes you get the non-ASCII single quote!